crates/fuel-core/src/coins_query.rs

```diff
@@ -48,7 +42,7 @@ pub enum CoinsQueryError {
     #[error("the target cannot be met due to no coins available or exceeding the {max} coin limit.")]
     InsufficientCoinsForTheMax {
         asset_id: AssetId,
-        collected_amount: Word,
+        collected_amount: u128,
```

Comment:
@xgreenx - why do we use u128 here? It breaks the client currently so I'll revert the change for now and we can re-introduce it later.